src/dashboard/components/Dashboard.test.tsx

```diff
@@ -51,24 +66,58 @@ describe('Dashboard', (): void => {
         ).not.toThrow();
     });
 
-    it('should render the no-submissions page if there are no submissions', async (): Promise<void> => {
-        const { findByText } = render(<Dashboard />, {
-            wrapper: combineWrappers(apolloWrapper(generateMockQueryResponse([])), routerWrapper(['/link-1'])),
+    describe('No Submissions', () => {
+        it('should render the no-submissions page if there are no submissions', async (): Promise<void> => {
+            const { container } = render(<Dashboard />, {
+                wrapper: combineWrappers(apolloWrapper(generateMockQueryResponse([])), routerWrapper(['/link-1'])),
+            });
+            await wait();
+            expect(container.querySelector('.no-submissions')).toBeInTheDocument();
+        });
+        it('should add a submission then display the full dashboard when start submission button clicked', async (): Promise<
```

Comment:
I spent some time writing this test then realised we're going to be changing this functionality to redirect to the article type page 🤷‍♂ 🙈 Doh!